com.github.taiko2k.avvie.json

```diff
@@ -0,0 +1,54 @@
+{
+   "app-id":"com.github.taiko2k.avvie",
+   "runtime":"org.gnome.Platform",
+   "runtime-version":"3.32",
+   "sdk":"org.gnome.Sdk",
+   "command":"run.sh",
+   "finish-args":[
+      "--share=ipc",
+      "--socket=wayland",
+      "--socket=fallback-x11",
+      "--filesystem=xdg-pictures",
+      "--filesystem=xdg-download",
+      "--talk-name=org.freedesktop.Notifications",
+      "--talk-name=org.freedesktop.FileChooser"
```

Comment:
Do you actually need these two?

Reply:
I want to have file **drag and drop** functionality. So I picked those two as a compromise in not exposing the whole host but common places the user might want to load file from. Also I have a **one click save** feature so I want to be able to write immediately to relevant locations.

Reply:
I see that talk-FileChooser isn't necessary in order to access the portal. My mistake, I've gone ahead and removed that one.

Reply:
:ok_hand: